perf/data/timbl.ttl

```diff
@@ -0,0 +1,348 @@
+@prefix c39: <https://janschill.net/profile/card#>.
+@prefix l: <https://www.w3.org/ns/iana/language-code/>.
+@prefix wd: <http://www.wikidata.org/entity/>.
+@prefix profi: <./>.
+@prefix inb0: </inbox/>.
+@prefix ti: </>.
+@prefix c36: <https://dimou.solidcommunity.net/profile/card#>.
+@prefix c40: <https://hindia.solidcommunity.net/profile/card#>.
+@prefix c42: <https://janikv.solidcommunity.net/profile/card#>.
+@prefix c53: <https://jenni.inrupt.net/profile/card#>.
+@prefix c54: <https://jyj.solidcommunity.net/profile/card#>.
+@prefix c: <https://vincentt.inrupt.net/profile/card#>.
+@prefix c0: <https://acoburn.inrupt.net/profile/card#>.
+@prefix c1: <https://csarven.inrupt.net/profile/card#>.
+@prefix c2: <https://davi.inrupt.net/profile/card#>.
+@prefix c3: <https://emmettownsend.inrupt.net/profile/card#>.
+@prefix c4: <https://jackson.solidcommunity.net/profile/card#>.
+@prefix c5: <https://jholleran.inrupt.net/profile/card#>.
+@prefix c6: <https://johnb.inrupt.net/profile/card#>.
+@prefix c9: <https://joshcollins.inrupt.net/profile/card#>.
+@prefix c10: <https://justin.inrupt.net/profile/card#>.
+@prefix c11: <https://kellyo.inrupt.net/profile/card#>.
+@prefix c12: <https://khoward.inrupt.net/profile/card#>.
+@prefix c14: <https://michielbdejong.inrupt.net/profile/card#>.
+@prefix c15: <https://nseydoux.inrupt.net/profile/card#>.
+@prefix c17: <https://oz.inrupt.net/profile/card#>.
+@prefix c18: <https://sharonstrats.inrupt.net/profile/card#>.
+@prefix c19: <https://solid.kjernsmo.net/profile/card#>.
+@prefix c20: <https://solid.zwifi.eu/profile/card#>.
+@prefix c21: <https://worrallp.inrupt.net/profile/card#>.
+
+occup:6f7c1e84-8162-4c20-a575-b57ff624eef5 schema:name "taxi driver".
+
+occup:7b1b5da8-573a-49bb-a38e-68725a949f4f schema:name "chief technology officer".
+
+occup:e3229e40-f571-4b26-baca-29edc8fe313e schema:name "computer scientist".
+
+occup:f2b15a0e-e65a-438a-affb-29b9d50b77d1 schema:name "software developer".
+
+skill:6c870993-9341-438b-91c0-c7fe4f96d8f5 schema:name "apply strategic thinking".
+
+skill:a72408af-b488-4ad4-801b-bbf16f7a7b57
+schema:name "install blinds drive systems".
+skill:a917fd91-866d-4ecf-a70e-894b755f9baf
+schema:name "develop mechatronic test procedures".
+skill:f28617ad-afdd-4041-814c-216153a38998
+schema:name "analyse software specifications".
+skill:f6868852-9bf6-4899-a5d6-5f9532fb877d
+schema:name "design component interfaces".
+wd:Q106746930 schema:name "Inrupt Inc."@en.
+
+wd:Q10853538 schema:name "".
+
+wd:Q16601665 schema:name "Principality of Masserano".
+
+wd:Q34433 schema:name "University of Oxford"@en.
+
+wd:Q42944 schema:name "CERN"@en.
+
+wd:Q49108 schema:name "Massachusetts Institute of Technology"@en.
+
+profi:card a foaf:PersonalProfileDocument; foaf:maker :me; foaf:primaryTopic :me.
+
+:id1618591080534
+    a solid:CurrentRole;
+    schema:description
+        "Technical strategic architecture for Inrupt. Leader of the Solid movement. Coordinate and contribute to Solid open source.";
```

Comment:
Inconsistent structure... Is this a list of nouns or verbs? Should be "Design technical & strategic architecture for Inrupt. Lead the Solid movement. Coordinate and contribute to Solid open source." or "Technical strategic architect for Inrupt. Leader of the Solid movement. Coordinator and contributor to Solid open source."